osgar/drivers/oak_camera.py

```diff
@@ -25,12 +25,24 @@ def cam_is_available(cam_ip):
     return False
 
 
+def get_video_encoder(name):
+    # https://docs.luxonis.com/projects/api/en/latest/components/nodes/video_encoder/
+    if name == 'h264':
+        return dai.VideoEncoderProperties.Profile.H264_MAIN
+    elif name == 'h265':
+        return dai.VideoEncoderProperties.Profile.H265_MAIN
+    elif name == 'mjpeg':
+        return dai.VideoEncoderProperties.Profile.MJPEG
+    else:
+        assert 0, f'"{name}" is not supported'
+
+
 class OakCamera:
     def __init__(self, config, bus):
         self.input_thread = Thread(target=self.run_input, daemon=True)
         self.bus = bus
 
-        self.bus.register('depth', 'color', 'orientation_list', 'detections',
+        self.bus.register('depth:gz', 'color', 'orientation_list', 'detections',
```

Comment:
This change increases the load on the cpu. In my opinion, this option (`:gz`) must be optional.

Reply:
Are you sure about it? Realsense also uses `:gz` version - I mean it will surely slow down CPU, but save time on storage??

Reply:
p.s. it would be interesting/nice, to do this packing already on OAK-D side, but then you spend CPU time unpacking if you use it

Reply:
I agree let this be the default setting. However, in one application in particular we are hitting a cpu performance limit and this change will probably make our work more difficult.
Thus, if possible, I would be happy for the option to turn this off.
Yes, the situation is the same with the RS camera, but we don't use it now.

Reply:
OK, thanks - if we agree that default should be gzipped and you already have application which would reach the limit, then let's add the on/off option.